README.md

```diff
@@ -112,7 +121,14 @@ In either case, you will want to have [Serverless] installed, eg. `npm install -
   You can test the function and see the invocation results with
   `sls invoke -f myfunc`.
 
-  To invoke the function locally, use `sls invoke local -f myfunc`.
+* To test locally, you can use `sls invoke local`
+
+  Note that this can take an hour to build, consider adding `export SLS_DEBUG=*` so you can see what serverless is doing.
```

Comment:
Can we change this to say that the _first_ build can be long because of a need for a Docker build?

Reply:
Done